package.json

```diff
@@ -16,8 +16,8 @@
     "watch:mobile": "cs watch --mobile",
     "start": "cs start --hot --browser",
     "deploy": "git-directory-deploy --directory build/ --branch ${DEPLOY_BRANCH:-build} --repo ${DEPLOY_REPOSITORY:-https://cozy-bot:$GITHUB_TOKEN@github.com/cozy/cozy-notes}",
-    "test": "cs test --verbose --coverage",
-    "cozyPublish": "cs publish --token $REGISTRY_TOKEN --build-commit $(git rev-parse ${DEPLOY_BRANCH:-build})"
+    "test": "cs test --verbose --coverage",   
+    "cozyPublish": "cozy-app-publish --token $REGISTRY_TOKEN --prepublish downcloud --space default --postpublish mattermost"
```

Comment:
It seems mattermost postpublish hook needs some environment variables: https://github.com/cozy/cozy-libs/tree/master/packages/cozy-app-publish#mattermost-postpublish-hook.

Look what we do on banks:
https://github.com/cozy/cozy-banks/blob/master/package.json#L17
https://github.com/cozy/cozy-banks/blob/master/.travis.yml#L66
https://github.com/cozy/cozy-banks/blob/master/.travis.yml#L70-L71